ansible/roles/schulcloud-client-core/templates/client-configmap-files.yml.j2

```diff
@@ -26,7 +26,7 @@ data:
       Use:
         defaultSrc, fontSrc, styleSrc, scriptSrc, imageSrc,
         connectSrc, mediaSrc, objectSrc, prefetchSrc, childSrc,
-        frameSrc, workerSrc, frameancestorsSrc, formactionSrc, baseuriSrc
+        frameSrc, workerSrc, frameAncestors, formactionSrc, baseuriSrc
```

Comment:
do we need to update also the reference to 
https://acc.bettermarks.com in line 65 below?

Reply:
no